editors/code/package.json

```diff
@@ -32,7 +32,8 @@
     },
     "dependencies": {
         "seedrandom": "^3.0.1",
-        "vscode-languageclient": "^5.3.0-next.4"
+        "vscode-languageclient": "^5.3.0-next.4",
+        "jsonc-parser": "^2.1.0"
```

Comment:
I have no clue, I will investigate.

Reply:
@matklad I looked into it, and although I couldn't get a clear answer on it, it seems to be used in a lot of VSCode extensions - that might indicate that perhaps there is no way to get this out of vscode it self .

It was also not a dependency in the npm package. 
That being said, perhaps there is a different way of getting values out of the themes. Will look into that to.

Reply:
@matklad Alright I added two changes in the latest commit.
[This](https://github.com/rust-analyzer/rust-analyzer/pull/2061/files#diff-00a4dd938fe20be620cc24ee008c7bf1R47-R56) changes the code imported from tree sitter to rely on vscode loading the extensions package json instead of doing it manually. 

However we're still parsing themes manually and there is no way around that for now. However it seems to work fine with JSON and not JSONC - I'm making the assumption that the themes I tested with do not use JSONC. 
Unless we get some issues with certain themes using JSONC, we can omit it for now - and only import it if someone reports an issue with their themes loading. 
This resolves this conversation.

Reply:
Resolved in https://github.com/rust-analyzer/rust-analyzer/pull/2061/commits/49562d36b97ddde34cf7585a8c2e8f232519b657